deployments/setup/setup-openchoreo.sh

```
@@ -408,8 +408,118 @@ echo "All core OpenChoreo planes are installed and registered!"
+echo ""
+
+# Default-env Thunder provisioning moved to the Makefile's setup-default-env-thunder
+# target: it now needs AMS up (store_via_ams), which this script runs before.
+
+# ============================================================================
+# Step 6: Install Observability Extension (Agent Manager Observer)
+# ============================================================================
+echo "6️⃣  Observability Extension (Agent Manager Observer)"
+if ! helm status wso2-amp-observability-extension -n openchoreo-observability-plane &>/dev/null; then
+    echo "Building and loading Agent Manager Observer Docker image into k3d cluster..."
+    make -C ${PROJECT_ROOT}/agent-manager-observer docker-load-k3d
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)